optimade/models/structures.py

```diff
@@ -214,485 +216,484 @@ class StructureResourceAttributes(EntryResourceAttributes):
     elements: List[str] = Field(
         ...,
         description="""Names of the different elements present in the structure.
+
 - **Type**: list of strings.
-- **Requirements/Conventions**:
 
-  - **Support**: SHOULD be supported, i.e., SHOULD NOT be :val:`null`. Is REQUIRED in this implementation, i.e., MUST NOT be :val:`null`.
-  - **Query**: MUST be a queryable property with support for all mandatory filter operators.
-  - The strings are the chemical symbols, written as uppercase letter plus optional lowercase letters.
-  - The order MUST be alphabetical.
-  - Note: This may not contain the "x" that is suggested in chemical_symbols for the :property:`species` property.
+- **Requirements/Conventions**:
+    - **Support**: SHOULD be supported by all implementations, i.e., SHOULD NOT be `null`.
+    - **Query**: MUST be a queryable property with support for all mandatory filter features.
+    - The strings are the chemical symbols, i.e., either a single uppercase letter or an uppercase letter followed by a number of lowercase letters.
+    - The order MUST be alphabetical.
+    - Note: This property SHOULD NOT contain the string "X" to indicate non-chemical elements or "vacancy" to indicate vacancies (in contrast to the field `chemical_symbols` for the `species` property).
 
 - **Examples**:
-
-  - :val:`["Si"]`
-  - :val:`["Al","O","Si"]`
+    - `["Si"]`
+    - `["Al","O","Si"]`
 
 - **Query examples**:
-  - A filter that matches all records of structures that contain Si, Al **and** O, and possibly other elements: :filter:`elements HAS ALL "Si", "Al", "O"`.
-  - To match structures with exactly these three elements, use :filter:`elements HAS ALL "Si", "Al", "O" AND LENGTH elements = 3`.""",
+    - A filter that matches all records of structures that contain Si, Al **and** O, and possibly other elements: `elements HAS ALL "Si", "Al", "O"`.
+    - To match structures with exactly these three elements, use `elements HAS ALL "Si", "Al", "O" AND elements LENGTH 3`.""",
     )
 
     nelements: int = Field(
         ...,
         description="""Number of different elements in the structure as an integer.
+
 - **Type**: integer
+
 - **Requirements/Conventions**:
+    - **Support**: SHOULD be supported by all implementations, i.e., SHOULD NOT be `null`.
+    - **Query**: MUST be a queryable property with support for all mandatory filter features.
 
-  - **Support**: SHOULD be supported, i.e., SHOULD NOT be :val:`null`. Is REQUIRED in this implementation, i.e., MUST NOT be :val:`null`.
-  - **Query**: MUST be a queryable property with support for all mandatory filter operators.
+- **Example**: `3`
```

Comment:
I would look nicer if it would be a list even there is only one example. I can see in the specification is the same so it is fine for me.
The specification doesn't seem to be very consistent in this either:
- sometimes there is `Examples` with a single example
- sometimes there is a single `Example` as a list

Reply:
Yeah I agree, maybe we can change it here and I'll add it to the formatting fixes PR I have open on the spec.

Reply:
Have changed all occurrences here and in the spec.